docs/sources/reference/api/docker_remote_api_v1.16.md

```diff
@@ -1326,6 +1327,11 @@ Create a new image from a container's changes
              "Cmd":[
                      "date"
              ],
+	     "Labels": [
+                     "Vendor=Acme",
+                     "License=GPL",
+                     "Version=1.0"
+             ],
```

Comment:
@phemmer I agree a map is better, I actuallly updated the code as such, but not the documentation yet.